qa/suites/orch/cephadm/workunits/task/test_nfs.yaml

```diff
@@ -1,17 +0,0 @@
-roles:
-- - host.a
-  - osd.0
-  - osd.1
-  - osd.2
-  - mon.a
-  - mgr.a
-  - client.0
-tasks:
-- install:
-- cephadm:
-- cephadm.shell:
-    host.a:
-      - ceph orch apply mds a
-- cephfs_test_runner:
-    modules:
-      - tasks.cephfs.test_nfs
```

Comment:
is it necessary to remove the test from the orch/cephadm suite as part of this? I get also wanting to have it in the fs suite, but this has caught some failures for us in the past. We have some tests that run in both the rados and orch/cephadm suite for similar reasons.

Reply:
OK. That was not the intent. I basically forgot to ask you the question in this PR since I had to rush out for some work. Sorry!

How do you want to do this? Have a symlink pointer from orch:cephadm:nfs to fs:nfs?

Reply:
I think a symlink like that works. As long as both suites are running this test_nfs task.